admin/services/data-access/sql.service.js

```diff
@@ -282,6 +282,23 @@ sqlService.findOneById = async (table, id) => {
   return R.head(rows)
 }
 
+/**
+ * Find a row by numeric pupil ID
+ * @param {string} table
+ * @param {number} pupilId
+ * @return {Promise<void>}
+ */
+sqlService.findOneByPupilId = async (table, pupilId) => {
```

Comment:
please remove this.  it belongs in a data service

Reply:
@GuyHarwood removed